src/bundle/Controller/ContentTypeController.php

```diff
@@ -148,14 +169,42 @@ public function addAction(ContentTypeGroup $group): Response
         ]);
     }
 
+    /**
+     * @param \eZ\Publish\API\Repository\Values\ContentType\ContentTypeGroup $group
+     * @param \eZ\Publish\API\Repository\Values\ContentType\ContentType $contentType
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException
+     * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException
+     */
     public function editAction(ContentTypeGroup $group, ContentType $contentType): Response
     {
+        // It is possible to have two user wants to edit the same Content Type. Due the Kernel it is not possible.
+        // So we need to catch 'BadStateException' and inform user about another user editing the Content Type
         try {
             $contentTypeDraft = $this->contentTypeService->loadContentTypeDraft($contentType->id);
             $this->contentTypeService->deleteContentType($contentTypeDraft);
             $contentTypeDraft = $this->contentTypeService->createContentTypeDraft($contentType);
         } catch (NotFoundException $e) {
-            $contentTypeDraft = $this->contentTypeService->createContentTypeDraft($contentType);
+            try {
+                $contentTypeDraft = $this->contentTypeService->createContentTypeDraft($contentType);
+            } catch (BadStateException $e) {
+                $userId = $contentType->modifierId;
+                $this->notificationHandler->error(
+                    $this->translator->trans(
+                        /** @Desc("Draft of the Content Type '%name%' already exists. It is owned by user with identifier '%userId%' and name '%userName%'") */
```

Comment:
I'd change the message it to be more clear to the user:
`Draft of the Content Type '%name%' already exists and is locked by '%userContentName%`.

Because:
- "owned" does not imply that it cannot be modified, "locked" does
- while for admins Id might be valuable, in general exposing numeric id is as bad as exposing `login`.
If `%userContentName%` cannot be fetched I'd rather go with more friendly message: "another user" (and it fits better as an ending of that sentence).

POV ping @webhdx @Nattfarinn